packages/blade/src/components/Chip/ChipGroup.stories.tsx

```diff
@@ -0,0 +1,463 @@
+};
+DefaultSelectedSingle.argTypes = {
+  defaultValue: {
+    options: ['Proprietorship', 'Public', 'Small Business'],
+    control: {
+      type: 'select',
+    },
+  },
+};
+
+export const DefaultMultiSelected = MultiSelectChipTemplate.bind({});
+DefaultMultiSelected.storyName = 'Uncontrolled Multiple Selection with Default Value';
+DefaultMultiSelected.args = {
+  defaultValue: ['Automated Payment Links'],
+  accessibilityLabel: 'Select other capabilities you are looking for from the options below',
+  selectionType: 'multiple',
+};
+DefaultMultiSelected.argTypes = {
+  defaultValue: {
+    options: [
+      'Automated Payment Links',
+      'Wallet on My App',
+      'Offer discounts, Pay Later & EMI options',
+    ],
+    control: {
+      type: 'multi-select',
+    },
+  },
+};
+
+const ControlledSingleSelectionTemplate: ComponentStory<typeof ChipGroupComponent> = ({
+  ...args
+}) => {
+  const chipValues = ['Proprietorship', 'Public', 'Small Business'];
+  const [value, setValue] = React.useState('Proprietorship');
+  return (
+    <Box display="flex" gap="spacing.5" flexDirection="column" minHeight="200px">
+      <Dropdown marginRight="spacing.4">
+        <DropdownButton size="small">Business Type</DropdownButton>
+        <DropdownOverlay>
+          <ActionList>
+            {chipValues.map((chipValue: string) => (
+              <ActionListItem
+                key={chipValue}
+                title={chipValue}
+                value={chipValue}
+                onClick={({ name }) => setValue(name)}
+                isSelected={value === chipValue}
+              />
+            ))}
+          </ActionList>
+        </DropdownOverlay>
+      </Dropdown>
+
+      <ChipGroupComponent
+        {...args}
+        selectionType="single"
+        value={value}
+        onChange={({ values }) => setValue(values[0])}
+      >
+        {chipValues.map((chipValue: string) => (
+          <ChipComponent key={chipValue} value={chipValue}>
+            {chipValue}
+          </ChipComponent>
+        ))}
+      </ChipGroupComponent>
+    </Box>
+  );
+};
+
+export const ControlledSingleSelection = ControlledSingleSelectionTemplate.bind({});
+ControlledSingleSelection.storyName = 'Controlled Single Selection';
+ControlledSingleSelection.args = {
+  accessibilityLabel: 'Choose one business type from the options below',
+  selectionType: 'single',
+};
+
+const ControlledMultiSelectionTemplate: ComponentStory<typeof ChipGroupComponent> = (args) => {
+  const chipValues = [
+    'Automated Payment Links',
+    'Wallet on My App',
+    'Offer discounts, Pay Later & EMI options',
+  ];
+  const [values, setValues] = React.useState(['Automated Payment Links']);
+  return (
+    <Box display="flex" gap="spacing.5" flexDirection="column" minHeight="200px">
+      <Dropdown marginRight="spacing.4">
+        <DropdownButton size="small">What other capabilities are you looking for?</DropdownButton>
+        <DropdownOverlay>
+          <ActionList>
```

Comment:
this shall have selection type multiple right?  it looks like this right now. @saurabhdaware is this a valid multiselect?
<img width="1158" alt="Screenshot 2023-08-29 at 11 04 48 AM" src="https://github.com/razorpay/blade/assets/11384858/b16456be-3092-4438-82b2-0b9b4d5f4116">

Reply:
Fixed